homeassistant/components/pi_hole/switch.py

```diff
@@ -51,38 +59,40 @@ class PiHoleSwitch(PiHoleEntity, SwitchEntity):
     """Representation of a Pi-hole switch."""
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the switch."""
         return self._name
 
     @property
-    def unique_id(self):
+    def unique_id(self) -> str:
         """Return the unique id of the switch."""
         return f"{self._server_unique_id}/Switch"
 
     @property
-    def icon(self):
+    def icon(self) -> str:
         """Icon to use in the frontend, if any."""
         return "mdi:pi-hole"
 
     @property
-    def is_on(self):
+    def is_on(self) -> bool:
         """Return if the service is on."""
-        return self.api.data.get("status") == "enabled"
+        if self.api.data.get("status") == "enabled":
+            return True
+        return False
 
-    async def async_turn_on(self, **kwargs):
+    async def async_turn_on(self, **kwargs: "dict[str, Any]") -> None:
```

Comment:
`**kwargs` is typed as `Any`.
```suggestion
    async def async_turn_on(self, **kwargs: Any) -> None:
```